src/main/java/seedu/weme/logic/commands/AddCommand.java

```diff
@@ -18,14 +18,13 @@
 
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a meme to weme. "
             + "Parameters: "
-            + PREFIX_NAME + "NAME "
+            + PREFIX_FILEPATH + "PATH "
             + "[" + PREFIX_DESCRIPTION + "DESCRIPTION] "
             + "[" + PREFIX_TAG + "TAG]...\n"
             + "Example: " + COMMAND_WORD + " "
-            + PREFIX_NAME + "John Doe "
+            + PREFIX_FILEPATH + "import/funny_meme.jpg "
```

Comment:
Might be better if we put a common directory name like `C:\Users\username\Downloads`?